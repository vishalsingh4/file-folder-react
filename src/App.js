import React from 'react';

import File from "./components/File";

const App = () => {
    return <File />;
};

export default App;