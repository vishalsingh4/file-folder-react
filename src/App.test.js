import { render, cleanup } from '@testing-library/react';

import App from './App';

describe('<App />', () => {
    afterEach(cleanup);

    it('should render App component', () => {
        const { asFragment, getByTestId } = render(<App />);
        const files = getByTestId('files');
        expect(files).toBeInTheDocument();
    })
})