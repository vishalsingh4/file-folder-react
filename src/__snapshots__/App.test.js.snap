// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render App component 1`] = `[Function]`;
